Packs/CommonPlaybooks/Playbooks/playbook-Block_URL_-_Generic_v2.yml

```diff
@@ -0,0 +1,1199 @@
+      description: ""
+    nexttasks:
+      "#none#":
+      - "34"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1610,
+          "y": 570
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "34":
+    id: "34"
+    taskid: 39802624-7a78-4ac4-8999-7735896a5fcd
+    type: condition
+    task:
+      id: 39802624-7a78-4ac4-8999-7735896a5fcd
+      version: -1
+      name: Is Sophos Integration Is Enabled?
+      description: Returns 'yes' if integration brand is available. Otherwise returns 'no'
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "#default#":
+      - "2"
+      "yes":
+      - "35"
+    scriptarguments:
+      brandname:
+        simple: Forcepoint
+    results:
+    - brandInstances
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1610,
+          "y": 700
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "35":
+    id: "35"
+    taskid: d3a1e7fd-3deb-4910-8567-c5bcb6363576
+    type: regular
+    task:
+      id: d3a1e7fd-3deb-4910-8567-c5bcb6363576
+      version: -1
+      name: Forcepoint - Add URL's to a specific category
+      description: "Append a specific category in Forcepoint with the Blocked URLs. \nYou can choose to add those to a default category (For more info - refer to https://www.forcepoint.com/product/feature/master-database-url-categories)\nor specify a custom/user-defined category."
+      script: Forcepoint|||fp-add-address-to-category
+      type: regular
+      iscommand: true
+      brand: Forcepoint
+    nexttasks:
+      "#none#":
+      - "2"
+    scriptarguments:
+      urls:
+        simple: ${Blocklist.URL}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1610,
+          "y": 915
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: 05ed149d-f3c8-46ab-818f-4327e57b1f59
+    type: condition
+    task:
+      id: 05ed149d-f3c8-46ab-818f-4327e57b1f59
+      version: -1
+      name: Is User Verification Is Required?
+      description: Checks if a manual user verification is needed before block
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "#default#":
+      - "37"
+      "yes":
+      - "30"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: ${input.UserVerification}
```

Comment:
There is no input named userVerification under Block URL - Generic v2.
Please add the input and avoid using plain text.